### DISCUSSION TITLE
I Cant Add Custom Repository

### DISCUSSION THREAD

**Discussion by b4ldrick** [nathanmarlor/foxess_modbus_charge_period_card#3] [Support]
I keep getting this error when trying to add custom repository. 

<Theme nathanmarlor/foxess_modbus_charge_period_card> Repository structure for v1.0.0 is not compliant

I've never come accross this before. Any idea why? I have followed the installation instructions to the letter several times.

**Comment by canton7** [comment]
What version of HACS are you using? Go to HACS -> About.

**Comment by b4ldrick** [comment]
1.32.1. On 27 Jul 2023, at 07:30, Antony Male ***@***.***> wrote:﻿
What version of HACS are you using? Go to HACS -> About.

**Reply by canton7** [reply]
D'oh, the category should be Lovelace, not Theme. Sorry! I'll fix the readme.

**Comment by b4ldrick** [comment]
Thanks I’ll try it later. On 27 Jul 2023, at 08:20, Antony Male ***@***.***> wrote:﻿
D'oh, the category should be Lovelace, not Theme. Sorry! I'll fix the readme.
